src/plugins/plot/spectralPlot/SpectralView.vue

```diff
@@ -0,0 +1,13 @@
+<template>
```

Comment:
All of the spectral view stuff needs to be removed from this PR. Keep it in a separate branch if you want to use it as the basis for ongoing work.